### PR TITLE
Pipe operator simplication

### PR DESCRIPTION
**Pull request by eksperimental**
I have removed |>  at the end of the lines,
this only happened twice and one is an example, and I think it's inconsistent.

also code has been simplified using |>, avoiding creating unnecessary variables.

the last fix deals with some formatting issues in Path test: space missing before == , and lines over 100 chars

### CODE REVIEW

File: lib/ex_unit/lib/ex_unit/case.ex [hunk 1]
```diff
@@ -289,8 +289,10 @@ defmodule ExUnit.Case do
             "\"use ExUnit.Case\" in the current module"
     end
 
-    tags = tags ++ Module.get_attribute(mod, :tag) ++ moduletag
-    tags = tags |> normalize_tags |> Map.merge(%{line: env.line, file: env.file})
+    tags =
+      tags ++ Module.get_attribute(mod, :tag) ++ moduletag
+      |> normalize_tags
```
Comment (ericmj) [inline lib/ex_unit/lib/ex_unit/case.ex]: Please add parenthesis around this line to make precedence clear.

**Comment by ericmj** [comment]
I think we said that we are not accepting whitespace changes. The `|>` changes are good because that's an actual guideline we have, specially important to fix it in docs.

File: lib/ex_unit/lib/ex_unit/case.ex (hunk 1 above)
Comment (eksperimental) [inline lib/ex_unit/lib/ex_unit/case.ex]: sorry I don't  quite get what you mean eric,
do you have an example to look at?

**Comment by eksperimental** [comment]
@ericmj i have updated the PR.

File: lib/ex_unit/lib/ex_unit/case.ex (hunk 1 above)
Comment (eksperimental) [inline lib/ex_unit/lib/ex_unit/case.ex]: oh.. do you mean around `tags ++ Module.get_attribute(mod, :tag) ++ moduletag` ?

**Comment by eksperimental** [comment]
I would also like to know if these guidelines that you just talked about are written somewhere, because I just learned them by written elixir code and reading comments by fellow developers when people submit PRs, just to draw the line between what is guideline and what is a style that we don't necessarily need to follow.

**Comment by ericmj** [comment]
The guidelines are only informal atm, based on conventions in the code base and what we have merged in the past.

**Comment by ericmj** [comment]
Thank you for the PR.

**Comment by eksperimental** [comment]
No worries eric

**Pull request merged**